webapi/Controllers/ServiceInfoController.cs

```diff
@@ -83,22 +92,24 @@ public IActionResult GetServiceInfo()
     [AllowAnonymous]
     public IActionResult GetAuthConfig()
     {
-        string authorityUriString = string.Empty;
-        if (!string.IsNullOrEmpty(this._chatAuthenticationOptions.AzureAd!.Instance) &&
-            !string.IsNullOrEmpty(this._chatAuthenticationOptions.AzureAd!.TenantId))
+        var azureAd = this._chatAuthenticationOptions.AzureAd!;
```

Comment:
How can you guarantee AzureAd will never be null?

It might be easier / safe to do the checks at line 97 and 98 on this._chatAuthenticationOtions.AzureAd and THEN assign var azureAd with its value...

Reply:
This is an artifact of the pre-existing auth work.  I haven't altered nullability assumptions or processing here.